Upstream's config does not autostart pipewire-media-session anymore, which
is an essential component. It's not easy to start it externally in a script
since it needs the pipewire socket up, and Void doesn't have a mechanism to
ensure it (systemd socket activation).
diff --git a/src/daemon/pipewire.conf.in.orig b/src/daemon/pipewire.conf.in
index 95632c2..ab0b097 100644
--- a/src/daemon/pipewire.conf.in.orig
+++ b/src/daemon/pipewire.conf.in
@@ -236,7 +236,7 @@
     # but it is better to start it as a systemd service.
     # Run the session manager with -h for options.
     #
-    @sm_comment@{ path = "@session_manager_path@" args = "@session_manager_args@" }
+    { path = "@session_manager_path@" args = "@session_manager_args@" }
     #
     # You can optionally start the pulseaudio-server here as well
     # but it is better to start it as a systemd service.
